tfhe/src/integer/server_key/radix_parallel/tests_unsigned/mod.rs

```diff
@@ -29,27 +29,13 @@ use rand::prelude::ThreadRng;
 use rand::Rng;
 use std::sync::Arc;
 
-/// Number of loop iteration within randomized tests
-#[cfg(not(tarpaulin))]
-pub(crate) const NB_TESTS: usize = 30;
-/// Smaller number of loop iteration within randomized test,
-/// meant for test where the function tested is more expensive
-#[cfg(not(tarpaulin))]
-pub(crate) const NB_TESTS_SMALLER: usize = 10;
-// Use lower numbers for coverage to ensure fast tests to counter balance slowdown due to code
-// instrumentation
-#[cfg(tarpaulin)]
-pub(crate) const NB_TESTS: usize = 1;
-#[cfg(tarpaulin)]
-pub(crate) const NB_TESTS_SMALLER: usize = 1;
-
 #[cfg(not(tarpaulin))]
 pub(crate) const NB_CTXT: usize = 4;
 #[cfg(tarpaulin)]
 pub(crate) const NB_CTXT: usize = 2;
 
 #[cfg(not(tarpaulin))]
-pub(crate) const NB_TESTS_UNCHECKED: usize = NB_TESTS;
+pub(crate) const NB_TESTS_UNCHECKED: usize = 30;
```

Comment:
Can't this be also be replaced by a function?

Reply:
Yes it probably can, Should it be done in this PR or one after ?

Reply:
I did it in this pr